timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -198,22 +122,14 @@ def __init__(
         else:
             self.must_drop_median = False
 
-        self._user_params, self._user_params_aux = check_and_split_hyperparameters(hyperparameters)
         self._oof_predictions: Optional[List[TimeSeriesDataFrame]] = None
 
-        self.params: Dict[str, Any] = {}
-        self.params_aux: Dict[str, Any] = {}
-        self._init_params_aux()
-        self._init_params()
-        self._is_initialized = True
-
-        # TODO: remove the variables below
-        self.nondefault_params: List[str] = []
+        self._hyperparameters, self._extra_ag_args = self.check_and_split_hyperparameters(hyperparameters)
```

Comment:
Nit: Maybe add a comment explaining that `self._hyperparameters` denotes the user-specified hyperparameters of the model? Alternatively, we can call this `self._user_hyperparameters`